src/axom/spin/internal/linear_bvh/build_radix_tree.hpp

```diff
@@ -464,13 +448,22 @@ void propagate_aabbs(RadixTree<FloatType, NDIMS>& data, int allocatorID)
 
   int32* counters_ptr = axom::allocate<int32>(inner_size, allocatorID);
```

Comment:
Do you have a sense for typical values of `data.inner_size` ?

Reply:
`inner_size` is set to `leaf_size - 1`